### ISSUE TITLE
Creating page results in "com.ctc.wstx.exc.WstxParsingException: Unexpected close tag </p>; expected </br>"

### ISSUE THREAD

**Issue by bentolor** [asciidoctor/asciidoctor-confluence#8]
Thanks for this tool! If I get `asciidoctor-confluence` to work, it would solve soo many issues for me.

Thing is: It works for a simple page, but on using the following minimal AsciiDoc fragment it will throw an error:

```
* foo
* bar +
* baz
```

which results in:

```
➜ doc git:(master) ✗ ./upload-to-confluence 
Enter Password: 
An error occurred, the page has not been created because:
[com.ctc.wstx.exc.WstxLazyException] com.ctc.wstx.exc.WstxParsingException: Unexpected close tag </p>; expected </br>.
 at [row,col {unknown-source}]: [16,26]
```

I assume that some party is whining about a non-closed `<br>`-Tag (which is perfectly fine for HTML).

Unfortunately my Ruby skills are void and I even have no clue where the troublesome code sits in.

**Comment by bentolor** [comment]
Same issue for images: `image:examples.png[MyExample,400,link="examples.png"]` will fail.

**Comment by Kabhal** [comment]
Hello
This is probably the html vs xhtml problem.

Try to add : ```:backend: xhtml``` to your adoc to force xhtml backend.

**Comment by bentolor** [comment]
Yes, adding the xhtml-slause seems to workaround this problem. Thank you!

I think the script should guide the user regarding this issue. If I understand the related issue, that change would fix it globally w/o requiring to add the clause to each of my documents. 

So please feel free to close this issue if appropriate.

**Comment by mojavelinux** [comment]
resolved by #26
